js/compiler/compiler.go

```diff
@@ -243,7 +243,9 @@ func newBabel() (*babel, error) {
 	return result, err
 }
 
-func increaseMappingsByOne(sourceMap []byte) ([]byte, error) {
+// this function increases the lines in the sourcemap by line so that it fixes the case where we need to wrap an
+// required file in a function to support/emulate commonjs
+func (c *compilationState) increaseMappingsByOne(sourceMap []byte) ([]byte, error) {
```

Comment:
```suggestion
// increaseMappingsByOne increases the lines in the sourcemap by line so that it fixes the case where we need to wrap a
// required file in a function to support/emulate commonjs
func (c *compilationState) increaseMappingsByOne(sourceMap []byte) ([]byte, error) {
```